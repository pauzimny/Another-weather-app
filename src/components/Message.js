import React from "react";
import Card from "@material-ui/core/Card";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles({
  msgCard: {
    backgroundColor: "#960c27",
    padding: "0 10px"
  }
});

const Message = ({ msg }) => {
  const classes = useStyles();

  return (
    <Card className={classes.msgCard}>
      <div className="message">{msg}</div>
    </Card>
  );
};

export default Message;
