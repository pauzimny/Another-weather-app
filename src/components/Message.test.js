import React from "react";
import Message from "./Message.js";
import { shallow, render, mount } from "enzyme";
import toJson from "enzyme-to-json";

it("renders correctly", () => {
  const wrapper = shallow(<Message msg={"You hava some message"} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});

it("renders correctly", () => {
  const wrapper = render(<Message msg={"You hava some message"} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});

it("renders correctly", () => {
  const wrapper = mount(<Message msg={"You hava some message"} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
