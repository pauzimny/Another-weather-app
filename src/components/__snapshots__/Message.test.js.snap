// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="message"
>
  You hava some message
</div>
`;

exports[`renders correctly 2`] = `
<div
  class="message"
>
  You hava some message
</div>
`;

exports[`renders correctly 3`] = `
<Message
  msg="You hava some message"
>
  <div
    className="message"
  >
    You hava some message
  </div>
</Message>
`;
